trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts

```diff
@@ -57,6 +57,11 @@ const getCheckboxIdForFeeRecordsAndStatus = (feeRecords: FeeRecord[], status: Fe
   return `feeRecordIds-${feeRecordIdList}-reportedPaymentsCurrency-${reportedPaymentsCurrency}-status-${status}`;
 };
 
+const getCheckboxAriaLabel = (feeRecords: FeeRecord[]): string => {
+  const feeRecordFacilityIdList = feeRecords.map(({ facilityId }) => facilityId).join(' ');
```

Comment:
Not sure on this but if we're joining facility ID's like this, is easily distinguishable in the screen reader audio output that they are separate numbers? As in if we have facility ID's 123 and 456, is it clear that these are not a single facility ID?

Reply:
It pauses between the two as if they are separate words, when I added columns it started reading them as 'one million eight hundred and thirty thousand.....' and so on which was I guess also ok, except it was inconsistent between ones where there is only one id which it decided to read as 'one eight three zero' so for consistency removed the commas.

Reply:
Cool thanks for clarifying